File: Java Programming/Competitive Programming/Coding Challenges 13/program2.java
//2. Accept amount in US dollar and return its corresponding value in Indian currency.
//Consider 1$ as 70 rupees.
//Output is:
// Input : 10
// Output : 700
// Input : 3
// Output : 210
// Input : 1200
// Output : 84000

import java.util.Scanner;

class program2
{
    static int DollarToINR(int iNo)
    {
        return iNo * 70;
    }

    public static void main(String[] args)
    {
        Scanner sobj = new Scanner(System.in);

        int iValue = 0;
        int iRet = 0;

        System.out.print("Enter number of USD: ");
        iValue = sobj.nextInt();

        iRet = DollarToINR(iValue);

        System.out.print("Value in INR is " + iRet);

        sobj.close();
    }
}

File: C++ Programming/Competitive Programming/Coding Challenges 13/program2.cpp
/*
2.Write a program which accept number from user and return the count of odd
digits.

Input : 2395
Output : 3

Input : 1018
Output : 2

Input : -1018
Output : 2

Input : 8462
Output : 0
*/

#include <iostream>
using namespace std;

int CountOdd(int iNo)
{
    int iCount = 0;

    if (iNo < 0)
    {
        iNo = -iNo;
    }

    for (; iNo != 0; iNo = iNo / 10)
    {
        int iDigit = iNo % 10;

        if (iDigit % 2 != 0)
        {
            iCount++;
        }
    }

    return iCount;
}

int main()
{
    int iValue = 0;
    int iRet = 0;

    cout << "Enter number : ";
    cin >> iValue;

    iRet = CountOdd(iValue);

    cout << "Count of odd digits is : " << iRet;

    return 0;
}